arch/arm/src/stm32/hardware/stm32_dma_v2.h

```diff
@@ -433,7 +433,7 @@
 #define DMAMAP_UART5_RX            STM32_DMA_MAP(DMA1,DMA_STREAM0,DMA_CHAN4)
 #define DMAMAP_USART3_RX           STM32_DMA_MAP(DMA1,DMA_STREAM1,DMA_CHAN4)
 #define DMAMAP_UART4_RX            STM32_DMA_MAP(DMA1,DMA_STREAM2,DMA_CHAN4)
-#define DMAMAP_USART3_TX_1         STM32_DMA_MAP(DMA1,DMA_STREAM3,DMA_CHAN4)
+#define DMAMAP_USART3_TX           STM32_DMA_MAP(DMA1,DMA_STREAM3,DMA_CHAN4)
```

Comment:
This is a breaking change that adds no real value. - Consider dropping it.

Reply:
Indeed, I've had an deeper look, DMAMAP_USART3_TX should have been set/chosen in board.h, my board configuration did not have that. Will revert this change.